security-release-process-documentation/security-release-process.md

```diff
@@ -286,3 +321,357 @@ Yes.
+- Modification of cluster or API objects without user consent in a common or 
+  default scenario
+
+##### Security features
+
+Breaking or bypassing any security feature provided. A vulnerability in a 
+security feature is rated “Important” by default, but the rating may be adjusted
+based on other considerations as documented here.
+
+Examples:
+
+- Disabling or bypassing a `NetworkPolicy` or `PodSecurityPolicy` without 
+  informing users or gaining consent
+- Reconfiguring a `NetworkPolicy` and allowing connections to other processes
+  without consent
+
+##### Information disclosure (targeted)
+
+Cases where the attacker can easily read information on the system from 
+specific locations, including system information, which was not 
+intended/designed to be exposed.
+
+Examples:
+
+- Targeted disclosure of the existence of a file
+- Targeted disclosure of anonymous data
+
+#### Moderate
+
+##### Denial of service
+
+- Remote Anonymous User 
+    - Temporary DoS _without_ amplification. Examples:
+        - Multiple remote clients consuming all available resources 
+          (sessions, memory) on a server by establishing sessions and 
+          keeping them open
+
+- Authenticated User
+    - Persistent DoS. Example:
+        - Logged in user can send a specific object and crash the API Server, 
+          and the crash is not due to a write access violation, exploitable 
+          or read access violation
+
+##### Spoofing
+
+An entity (computer, server, user, process) is able to masquerade as a 
+different, random entity that cannot be specifically selected.
+
+Examples:
+- Client properly authenticates to server, but server hands back a session 
+  from another random user who happens to be connected to the server at the 
+  same time
+
+##### Tampering
+
+Permanent or persistent modification of any user or system data in a specific scenario
+
+Examples:
+
+- Modification of application data files or databases in a specific scenario, 
+  for example writing to etcd when a user adds a specific type of API object and
+  only if the API version is v2.1
+- Modification of cluster or API objects without user consent in a specific scenario
+
+##### Security assurances
+
+A security assurance is either a security feature or another product 
+feature/function that customers expect to offer security protection. 
+Communications have messaged (explicitly or implicitly) that customers can 
+rely on the integrity of the feature, and that’s what makes it a security 
+assurance. Security bulletins will be released for a shortcoming in a security 
+assurance that undermines the customer’s reliance or trust.
+
+Examples:
+
+- Processes running with normal “user” privileges cannot gain “admin” 
+  privileges unless admin password/credentials have been provided 
+  via intentionally authorized methods.
+
+#### Low
+
+##### Tampering
+
+Temporary modification of data in a specific scenario that does not persist.
+
+### Client
```

Comment:
I think there are 2 classes of client vulnerabilities that are worth differentiating:

1. user interaction is required, but the attack target is remote (in the cluster) - examples include a dashboard vulnerability, or an XSS that can leverage an authenticated user's credentials when used with `kubectl proxy`
2. the target is a user or user's computer - for example the `kubectl cp` vulnerability

Reply:
+1 to this!